var/spack/repos/builtin/packages/sollve/package.py

```diff
@@ -0,0 +1,242 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Sollve(CMakePackage):
+    """The SOLLVE Project aims at scaling OpenMP by leveraging LLVM for exascale
+       performance and portability of applications.  This package provides a
+       collection of Clang/LLVM compilers and an OpenMP runtime library.
+    """
+
+    homepage = 'https://www.bnl.gov/compsci/projects/SOLLVE/'
+    url      = "https://github.com/SOLLVE"
```

Comment:
You can get rid of this `url` since it isn't used. You can actually move the `git` up here like so:
```suggestion
    git      = "https://github.com/SOLLVE/llvm.git"
```
Then it won't need to be specified in every `version()` directive below.

Reply:
Thanks! You're right. I updated `urlt` and `git` in `version()` as you suggested.